lvm.conf

```diff
@@ -0,0 +1,2455 @@
+# This is an example configuration file for the LVM2 system.
+# It contains the default settings that would be used if there was no
+# /etc/lvm/lvm.conf file.
+#
+# Refer to 'man lvm.conf' for further information including the file layout.
+#
+# Refer to 'man lvm.conf' for information about how settings configured in
+# this file are combined with built-in values and command line options to
+# arrive at the final values used by LVM.
+#
+# Refer to 'man lvmconfig' for information about displaying the built-in
+# and configured values used by LVM.
+#
+# If a default value is set in this file (not commented out), then a
+# new version of LVM using this file will continue using that value,
+# even if the new version of LVM changes the built-in default value.
+#
+# To put this file in a different directory and override /etc/lvm set
+# the environment variable LVM_SYSTEM_DIR before running the tools.
+#
+# N.B. Take care that each setting only appears once if uncommenting
+# example settings in this file.
+
+
+# Configuration section config.
+# How LVM configuration settings are handled.
+config {
+
+	# Configuration option config/checks.
+	# If enabled, any LVM configuration mismatch is reported.
+	# This implies checking that the configuration key is understood by
+	# LVM and that the value of the key is the proper type. If disabled,
+	# any configuration mismatch is ignored and the default value is used
+	# without any warning (a message about the configuration key not being
+	# found is issued in verbose mode only).
+	# This configuration option has an automatic default value.
+	# checks = 1
+
+	# Configuration option config/abort_on_errors.
+	# Abort the LVM process if a configuration mismatch is found.
+	# This configuration option has an automatic default value.
+	# abort_on_errors = 0
+
+	# Configuration option config/profile_dir.
+	# Directory where LVM looks for configuration profiles.
+	# This configuration option has an automatic default value.
+	# profile_dir = "/etc/lvm/profile"
+}
+
+# Configuration section devices.
+# How LVM uses block devices.
+devices {
+
+	# Configuration option devices/dir.
+	# Directory in which to create volume group device nodes.
+	# Commands also accept this as a prefix on volume group names.
+	# This configuration option is advanced.
+	# This configuration option has an automatic default value.
+	# dir = "/dev"
+
+	# Configuration option devices/scan.
+	# Directories containing device nodes to use with LVM.
+	# This configuration option is advanced.
+	# This configuration option has an automatic default value.
+	scan = [ "/dev", "/.bottlerocket/rootfs/dev" ]
```

Comment:
This file was copied from the cloud repo. I think the only line changed from a default lvm.conf is this one.